marketredesign/mrd-auth0-laravel
# Initial version

## Proposed changes
Implements bare minimum implementation to add login and logout routes to any Laravel application.

## Types of changes

What types of changes does your code introduce to this repository?


## Testing
As always

### Unit testing
Phpunit

### Manual testing
* Add this as dependency to a Laravel installation.
* Use Auth0 login provider
* Verify login and logout works.

## Further comments
n/a
